index.html

```diff
@@ -15,14 +15,254 @@
+    <!-- end feat: banner -->
+      
+    <!-- END NAV AND BANNER FEAT#12, BUILD BY ZUBAIR -->
 
+    <!-- START ABOUT FEAT#13, BUILT BY MARCIN WICHA-->
+    <!-- start feat: why choose lunatic -->
+    <div class="feat-choose">
+      <div class="container">
+        <div class="row section-features">
+          <!-- start title -->
+          <div class="col-12">
+            <h2 class="feature-title">Why Choose Lunatic</h2>
+          </div>
+          <!-- end title -->
+        </div>
+        <!-- end feat: why choose lunatic -->
+        <!-- start feat: feature cards-->
+        <div class="row feature-cards-container justify-content-around">
+          <div class="col-md-4 col-12 feature-card-1 wow fadeInUp" data-wow-delay="0.3s">
+            <!-- start card 1 -->
+            <div class="feature-card">
+              <div class="feature-card-icon">
+                <img src="img/service-icon-1.png"/>
+              </div>
+              <h3>Chat With Groups</h3>
+              <p>Writers and stars of Veep have responded incredulously to the news an Australian politician required stitches way.</p>
+            </div>
+            <!-- end card 1 -->
+          </div>
+          <div class="col-md-4 col-12 feature-card-2 wow fadeInUp" data-wow-delay="0.5s">
+            <!-- start card 2 -->
+            <div class="feature-card">
+              <div class="feature-card-icon">
+                <img src="img/service-icon-2.png"/>
+              </div>
+              <h3>Globally Chat</h3>
+              <p>Writers and stars of Veep have responded incredulously to the news an Australian politician required stitches way.</p>
+            </div>
+            <!-- end card 2 -->
+          </div>
+          <div class="col-md-4 col-12 feature-card-3 wow fadeInUp" data-wow-delay="0.7s">
+            <!-- start card 3 -->  
+            <div class="feature-card">
+              <div class="feature-card-icon">
+                <img src="img/service-icon-3.png"/>
+              </div>
+              <h3>Totally Free</h3>
+              <p>Writers and stars of Veep have responded incredulously to the news an Australian politician required stitches way.</p>
+            </div>
+            <!-- end card 3-->
+          </div>
+        </div>
+      </div>
+    </div>
+    <!-- end feat: feature cards -->
+    <!-- END ABOUT FEAT#13, BUILT BY MARCIN WICHA-->
+    
+    <!-- START ABOUT FEAT#14, BUILT BY MARCIN WICHA-->
+    <!-- START SOCIAL AND HEIGH QUALITY CONTAINER -->
+    <div class="container">
+      <div class="row section-social-features">
+        <!-- start social features photo -->
+        <div class="col-sm-6 d-none d-md-block social-photo">
+          <img class="img-fluid" src="img/mobile-1.png" alt="Photograph of a mobile phones social network features."/>
+        </div>
+        <!-- end social features photo -->
+        <!-- start social features column -->
+        <div class="col-12 col-sm-6 social-feature-col">
```

Comment:
Bug: Content should resize on medium screen.
```suggestion
        <div class="col-12 col-md-6 social-feature-col">
```